.github/CONTRIBUTING.md

```diff
@@ -27,68 +27,52 @@ Please see our [Contributor Code of Conduct](https://github.com/ionic-team/ionic
 
 * We appreciate you taking the time to contribute! Before submitting a pull request, we ask that you please [create an issue](#creating-an-issue) that explains the bug or feature request and let us know that you plan on creating a pull request for it. If an issue already exists, please comment on that issue letting us know you would like to submit a pull request for it. This helps us to keep track of the pull request and make sure there isn't duplicated effort.
 
-* Looking for an issue to fix? Make sure to look through our issues with the [help wanted](https://github.com/ionic-team/ionic/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label!
+* Looking for an issue to fix? Make sure to look through our issues with the [type: bug](https://github.com/ionic-team/ionic/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+bug%22) label!
 
 ### Setup
 
 1. Fork the repo.
 2. Clone your fork.
-3. Make a branch for your change.
+3. Create a new branch from master for your change.
 4. Run `npm install` (make sure you have [node](https://nodejs.org/en/) and [npm](http://blog.npmjs.org/post/85484771375/how-to-install-npm) installed first)
+5. Repeat step 4 for every package (core / angular / ...)
```

Comment:
```suggestion
```

Why don't we directly add the install scripts in the main `package.json`? Something like this:
```
  "scripts": {
    "install:core": "cd ./core && npm install",
    "install:angular": "cd ./angular && npm install",
    "install": "npm run install:core && npm run install:angular",
    ...
  },
```